test/shopify_app/carrier_service_verification_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+require 'action_controller'
+require 'action_controller/base'
+
+class CarrierServiceVerificationController < ActionController::Base
+  include ShopifyApp::CarrierServiceVerification
+  def carts_update
```

Comment:
I can, but again, I was just copying from webhooks. Until we refactor, wouldn't it make sense to just keep them the same?